Src/FluentAssertions/Xml/XDocumentAssertions.cs

```diff
@@ -344,6 +344,219 @@ public XDocumentAssertions(XDocument document)
         return new AndWhichConstraint<XDocumentAssertions, IEnumerable<XElement>>(this, xElements);
     }
 
+    /// <summary>
+    /// Asserts that the <see cref="XDocument.Root"/> of the current <see cref="XDocument"/> doesn't have the specified child elements.
```

Comment:
🤔 Why are you consistently using "child elements" (as in plural)?

Reply:
Good catch! :)